bouncer/version.go

```diff
@@ -1,3 +0,0 @@
-package bouncer
-
-const Version = "1.14.0"
```

Comment:
This was not very useful. It's the CLI version, which is optional so that's why it's removed.